diff --git a/src/cmd/init.rs b/src/cmd/init.rs
index 2c6415b..f15fb96 100644
--- a/src/cmd/init.rs
+++ b/src/cmd/init.rs
@@ -74,9 +74,9 @@ pub fn execute(args: &ArgMatches) -> Result<()> {
     if let Some(author) = get_author_name() {
         debug!("Obtained user name from gitconfig: {:?}", author);
         config.book.authors.push(author);
-        builder.with_config(config);
     }
 
+    builder.with_config(config);
     builder.build()?;
     println!("\nAll done, no errors...");
 
